packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -204,15 +204,19 @@ class AppLayout extends ElementMixin(
           right: auto;
           bottom: var(--vaadin-app-layout-navbar-offset-bottom, var(--vaadin-viewport-offset-bottom, 0));
           left: var(--vaadin-app-layout-navbar-offset-left, 0);
-          transition: transform var(--vaadin-app-layout-transition);
+          transition: transform var(--vaadin-app-layout-transition), visibility var(--vaadin-app-layout-transition);
           transform: translateX(-100%);
           max-width: 90%;
           width: 16em;
           box-sizing: border-box;
           padding: var(--safe-area-inset-top) 0 var(--safe-area-inset-bottom) var(--safe-area-inset-left);
+          /* The drawer should be inaccessible by the tabbing navigation when it is closed. */
+          visibility: hidden;
```

Comment:
I think we should add explicit unit tests for `visibility` CSS property. We generally avoid doing that, but you can find some examples of tests like this in `vaadin-form-layout` and `vaadin-split-layout`.

Reply:
Good point, especially since it is an essential part of the accessibility fix.

Reply:
Added, but I would note that the `visibility` property is tested by the keyboard tests as well.